pkg/jobs/hive/hive.go

```diff
@@ -93,10 +114,16 @@ func monitorDeployStatus(client clientv1.Client, hiveset hiveclient.Interface, c
 			context.TODO(), clusterName, v1.GetOptions{})
 
 		if err = utils.LogError(err); err != nil {
+
+			// If the cluster deployment is already gone
+			if jobType == utils.Destroying {
+				klog.Warning("No cluster deployment for " + clusterName + " was found")
```

Comment:
expecting no clusterdeployment , shall we add ` errors.IsNotFound(err)` to make sure

Reply:
Agreed. Added the error check to better catch errors.